libs/island-ui/core/src/lib/AsyncSearch/AsyncSearch.tsx

```diff
@@ -57,6 +57,7 @@ export interface AsyncSearchProps {
   errorMessage?: string
   hasError?: boolean
   white?: boolean
+  isCompanySearch?: boolean
```

Comment:
This seems really specific. If I'm understanding this correctly, you are applying different styles to the input based on this prop, right? Are the new styles based on the design system?

I'd really like some more clarity on this 🙏☺️

Reply:
This is for an old component that I'm reusing (I made it myself years ago). But it seems it was never used next to a normal input field. So the style of the async search is completely different than a input field and it seemed really wonky. So this would compliment rest of the styles used in the application system. (CompanySearch is used in a few application in the application system, hence the name)

Reply:
@oddsson